beakerlib/index.js

```diff
@@ -0,0 +1,87 @@
+exports.parseProcedureTable = parseProcedureTable;
+function parseProcedureTable(val) {
```

Comment:
Readability-wise, I would prefer using oop style here, with `static #parse` and `#print` method.

Reply:
Done.